nginx/config.go

```diff
@@ -48,7 +48,8 @@ http {
 
 	client_max_body_size {{ $routerConfig.BodySize }};
 
-	set_real_ip_from {{ $routerConfig.ProxyRealIPCIDR }};
+	{{ range $realIPCIDR := $routerConfig.ProxyRealIPCIDRs }}set_real_ip_from {{ $realIPCIDR }};{{ end }}
+	real_ip_recursive on;
```

Comment:
Ah... I am.  (Not that I need to be.)  What I was _trying_ to do was avoid an unnecessary newline either before or after all the `set_real_ip_from`, but the consequence of that might have been that I don't get newlines _between_ them either.  I know you said this is non-blocking, but I do want to give this a look before I consider merging it.